src/thx/Arrays.hx

```diff
@@ -622,6 +622,21 @@ It returns the last element of the array or null if the array is empty.
     return array[array.length-1];
 
 /**
+Static wrapper for `Array` `map` function.
+**/
+  #if js inline #end
```

Comment:
Added a test here: https://github.com/fponticelli/thx.benchmark/blob/master/demo/MapVsNativeMap.hx

In nodejs I get:

```
┃ description ┃    ops/sec ┃  error ┃ samples ┃ performance ┃
┃━━━━━━━━━━━━━┃━━━━━━━━━━━━┃━━━━━━━━┃━━━━━━━━━┃━━━━━━━━━━━━━┃
┃ native map  ┃  3,252,165 ┃ ±1.48% ┃      46 ┃  80% slower ┃
┃ static map  ┃ 16,648,575 ┃ ±1.17% ┃      42 ┃     fastest ┃
```

Quite a difference.